contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -1037,6 +1048,9 @@ contract VaultHub is PausableUntilWithRoles {
         address _recipient,
         uint256 _amount
     ) internal {
+        uint256 totalValue_ = _totalValue(_record);
+        if (_amount > totalValue_) revert WithdrawAmountExceedsTotalValue(totalValue_, _amount);
```

Comment:
please update the `_rebalance` method accordingly (get rid of the duplicate code)